### DISCUSSION TITLE
Extenciones para VScode

### DISCUSSION THREAD

**Discussion by YerValMoCa** [Launch-X-Latam/MisionFrontEnd#172] [Ideas]
0-Auto Close Tag-creador: Jun Han, sirve para = escribir mas rapido.
1-Auto Rename Tag-creador: Jun Han, sirve para = completar mas rapido simbolos y palabras.
2-Color Highlight- Creador: Sergii N, sirve para = dar color a los simbolos y etiquetas.
3-Electron Highligter Syntax-creador: Mike McBride, sirve para = dar color a las etiquetas y variables.
4-ESLint-creador: Microsft, sirve para = trabajar java scripts en español.
5-Live Server-creador: Ritwick Dey, sirve para = trabajar en tiempo real con el navegador y actualizar los cambios en tiempo real cada vez que se guarda en el archivo "control + S".
6-Material Icon Theme-creador: Philipp Kief, sirve para = dar imagen a las carpetas y archivos.
7-Prettier - Code formatter-creador:Prettier, sirve para = dar color a los elementos.
8-Spanish Languaje Pack for Visual Studio Code-creador: Microsoft, sirve para = poner en español el visual estudio code.
9-vscode-icons-creador: VSCode Icons Team, sirve para = dar color a los iconos.
10-JavaScripts (ES6) code snippets-creador: charalamposkarypidis, sirve para = trabajar en español con java scripts.
Hay mas, aun haci con esas tienen y sirve para mucho. A la vez que el vscode de va añadiendo y recomendando aplicaciones segun tu configuracion de trabajo, en el camino con estas tendras mucho aporte y te ubicaras mas rapido con los archivos.

**Comment by cavazte** [comment]
Qué buen aporte! Gracias!

**Comment by maxkario** [comment]
Debo mencionar que hoy tuve un problema con la extensión Live Server, si a algunos no les llega a funcionar, la solución está en instalar la versión anterior a la actual.
